Prox/Prox/PlaceCarousel/PlaceCarouselHeaderView.swift

```diff
@@ -13,27 +13,30 @@ class PlaceCarouselHeaderView: HorizontalLineView {
         return label
     }()
 
+    fileprivate lazy var line: HorizontalLineView = HorizontalLineView()
+
     override init(frame: CGRect) {
         super.init(frame: frame)
         self.backgroundColor = .white
-        self.startX = 20.0
-        
-        self.setupViews()
-    }
 
-    // setting up the constraints for the views
-    func setupViews() {
-        color = Colors.carouselViewHeaderHorizontalLine
+        line.startX = 20.0
+        line.color = Colors.carouselViewHeaderHorizontalLine
+        line.backgroundColor = .clear
```

Comment:
nit: it could be a little confusing that you have to set the background to .clear on HorizontalLineView if you want to use it as a line each time.

Add a comment to the class? Or set .clear as a default value somewhere.